app/views/reports/_daily.haml

```diff
@@ -32,7 +32,7 @@
         %tr
           - klass = [cycle('even', 'odd')]
           - klass << 'post-ignored' if ignored?(post)
-          - replies_today = post.replies.where(created_at: @day.beginning_of_day .. @day.end_of_day).order('created_at asc')
+          - replies_today = post.replies.where(created_at: @day.beginning_of_day .. @day.end_of_day).ordered
```

Comment:
This is technically more of a `created_at` thing (at least we figure out how we're gonna deal with non-monotonic `created_at` timestamps), so I'd leave it as it was before. (It's used for a "last in day" link, and to say when the post was last updated, so it seems to make sense to have it be the `created_at` and appear in a later audit.)

Reply:
… then again there are many other places where we used `id asc` assuming the same thing and so we're gonna need to fiddle with stuff there. ugh. uh. yeah. idk, leave it for now probably.

Reply:
What Throne said; for report purposes, we do actually want creation time here.